mne/bem.py

```diff
@@ -1115,6 +1119,10 @@ def read_bem_solution(fname, verbose=None):
     -------
     bem : dict
         The BEM solution.
+
+    See Also
+    --------
+    write_bem_solution
```

Comment:
btw, I was trying to figure out with @wronk what's the difference between `make_bem_model` and `make_bem_solution`. Could you throw some light?